hivemind/averaging/averager.py

```diff
@@ -722,7 +724,8 @@ async def _load_state_from_peers(self, future: MPFuture, timeout: Optional[float
                             logger.debug(f"Peer {peer} did not send its state")
                             continue
 
-                        logger.info(f"Finished downloading state from {peer}")
+                        t1 = time.time()
```

Comment:
For measuring time intervals, it's best to use [time.monotonic](https://docs.python.org/3/library/time.html#time.monotonic) to make sure the intervals are non-negative

Reply:
done